src/Resources/views/CRUD/display_enum.html.twig

```diff
@@ -10,5 +10,9 @@ file that was distributed with this source code.
 #}
 
 {%- apply spaceless %}
-    {{ value.name }}
+    {%- if value is null -%}
+        &nbsp;
+    {%- else -%}
```

Comment:
isnt it better to just add an if? the else part where it is null can be omitted.

Reply:
I've follow what has been done in similar template (currency, percent, etc..) for coherency, but i can make the change requested, your call!